formulae/viewer/routes.py

```diff
@@ -0,0 +1,205 @@
+                                                         url=dict(), town=town, codex=codex, current_folios=current_folios,folios=folios)
+            images = formulae['images']
+            folios = formulae['folios']
+            town = formulae['town']
+            codex = formulae['codex']
+
+            #avoid error when the number of view is over the maximum
+            if view > len(images)-1:
+
+                flash( _('There are not {} images for this formula. Showing the last page image instead.'.format(view)))
+                view = len(images)-1
+                if "/images/formulae-1" in str(images[view]):
+                    link_picture=current_app.IIIFserver +str(images[view])
+                else:
+                    link_picture=str(images[view])
+                return current_app.config['nemo_app'].render(template='viewer::multiviewer.html', picture=link_picture, objectId=objectId,
+                                                         current_view=view, total_views=len(images), passage_data=passage_data,
+                                                         url=dict(), town=town, codex=codex, folios=folios)
+
+            #avoid error when the number of view is under the mainimum
+            elif view<0:
+                view=0
+                flash( _('There are not {} images for this formula. Showing the first page image instead.'.format(view)))
+                if "/images/formulae-1" in str(images[view]):
+                    link_picture=current_app.IIIFserver +str(images[view])
+                else:
+                    link_picture=str(images[view])
+                return current_app.config['nemo_app'].render(template='viewer::multiviewer.html', picture=link_picture,
+                                                             objectId=objectId, current_view=view, total_views=len(images),
+                                                             passage_data=passage_data,url=dict(), town=town,
+                                                             codex=codex, folios=folios)
+
+            current_folios = folios[view]
+
+            #create the link if it's an image in local server or take the complet link
+            if "/images/formulae-1" in str(images[view]):
+                link_picture=current_app.IIIFserver +str(images[view])
+            else:
+                link_picture=str(images[view])
+
+            return current_app.config['nemo_app'].render(template='viewer::multiviewer.html', picture=link_picture, objectId=objectId,
+                                                         current_view=view, total_views=len(images), passage_data=passage_data,
+                                                         url=dict(), town=town, codex=codex, current_folios=current_folios, folios=folios)
+    else:
+        flash( _('This corpus is on copyright, please choose another text'))
```

Comment:
Original messages should be in German. I would suggest:

`flash(_('Diese Formelsammlung ist noch nicht frei zugänglich.'))`